lib/tmdb/client.rb

```diff
@@ -142,6 +146,56 @@ def get_common_actors_between_movies(movie_one_title, movie_two_title)
         end
       end
 
+      def get_common_movies_between_multiple_actors(actor_names: nil, paginate_actor_names: nil, page: nil, sort_by: nil)
+        page = page.presence || 1
+        sort_by = sort_by.presence || 'popularity'
+        names = actor_names.uniq.reject { |name| name == '' }.compact.presence || paginate_actor_names.presence.split(';')
+        return if names.blank?
+
+        not_found_messages = []
+        person_ids = []
+        actor_names = []
+
+        actor_results ||= names.compact.each do |name|
```

Comment:
🤔 is `actor_results` being used anywhere?

Reply:
Oh yeah! 🤦‍♀️  Great catch! This is a relic of a previous version before I switched to hoisting those arrays. I'll remove it.